src/plugins/paginate/_base.scss

```diff
@@ -0,0 +1,31 @@
+.wb-paginate-pager {
```

Comment:
Is there some pagination style that exist in the data table CSS which will be preferable to be moved here?

Reply:
If both are render the same, then leave it as is for now.